indymeet/templates/puput/entry_page.html

```diff
@@ -43,14 +43,14 @@
 
 {% block content %}
         <div class="container mx-auto flex justify-center">
-            <article class="box max-w-4xl blog-container"
+            <article class="box max-w-5xl blog-container"
```

Comment:
if we wanted it the same as the rest of the pages, we'd have w-3/4 instead of max-w-5xl, but it feels too big here especially with our smaller font etc.